### PR TITLE
Add support for branding/re-theming

### PR DESCRIPTION
**Pull request by dangunter**
See #115

### CODE REVIEW

**Comment by dangunter** [comment]
I don't know the final color of the PROMMIS logo. Currently this looks kinda bad, colorwise:

image

**Comment by dangunter** [comment]
Here are the 3 variations as of the last push:

ui_branding_20240510

**Comment by dangunter** [comment]
Updated IDAES logo (white inside)

ui_branding_20240512

File: electron/ui/src/views/FlowsheetConfig/ConfigOutput/ConfigOutput.js [hunk 1]
```diff
@@ -55,43 +59,6 @@ export default function ConfigOutput(props) {
...
-            const href = window.URL.createObjectURL(data);
-            const link = document.createElement('a');
-            link.href = href;
-            link.setAttribute('download', 'sweep_results.csv');
-            document.body.appendChild(link);
-            link.click();
-            document.body.removeChild(link);
-        });
-    }
-
```
Comment (MichaelPesce) [inline electron/ui/src/views/FlowsheetConfig/ConfigOutput/ConfigOutput.js:94]: not sure if I'm missing something, are these download options being removed?

Comment (dangunter) [reply electron/ui/src/views/FlowsheetConfig/ConfigOutput/ConfigOutput.js]: I did somehow drop it! Thanks for catching that. It is re-inserted now, with some refactoring included to break out the different download sections into their own components.

**Comment by dangunter** [comment]
@MichaelPesce Finally! The test pass. Please have a look when you have a chance. But I think this is "done".

**Pull request merged**